examples/pack_quantized_rtn/run_all.sh

```
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+source /home/HDCharles/rhdev/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /home/HDCharles/rhdev/bin/activate was not specified as input (see shellcheck -x).

(SC1091)